cli/schemas/config-file.v1.json

```diff
@@ -414,6 +414,94 @@
           "type": "boolean",
           "default": true
         },
+        "quoteProps": {
+          "description": "Change when properties in objects are quoted in JavaScript and TypeScript.",
+          "default": "preserve",
+          "enum": ["asNeeded", "consistent", "preserve"]
+        },
+        "newLineKind": {
+          "description": "The newline character to use.",
+          "default": "lf",
+          "enum": ["auto", "crlf", "lf", "system"]
+        },
+        "useBraces": {
+          "description": "Whether to use braces for if statements, for statements, and while statements in JavaScript and TypeScript.",
+          "default": "whenNotSingleLine",
+          "enum": ["maintain", "whenNotSingleLine", "always", "preferNone"]
+        },
+        "bracePosition": {
+          "description": "The position of opening braces for blocks in JavaScript and TypeScript.",
+          "default": "sameLineUnlessHanging",
+          "enum": [
+            "maintain",
+            "sameLine",
+            "nextLine",
+            "sameLineUnlessHanging"
+          ]
+        },
+        "singleBodyPosition": {
+          "description": "The position of the body in single body blocks in JavaScript and TypeScript.",
+          "default": "sameLineUnlessHanging",
+          "enum": [
+            "maintain",
+            "sameLine",
+            "nextLine",
+            "sameLineUnlessHanging"
+          ]
+        },
+        "nextControlFlowPosition": {
+          "description": "Where to place the next control flow within a control flow statement in JavaScript and TypeScript.",
+          "default": "sameLine",
+          "enum": [
+            "maintain",
+            "sameLine",
+            "nextLine"
+          ]
+        },
+        "trailingCommas": {
+          "description": "Whether to add trailing commas in JavaScript and TypeScript.",
+          "default": "onlyMultiLine",
+          "enum": ["never", "always", "onlyMultiLine"]
+        },
+        "operatorPosition": {
+          "description": "Where to place the operator for expressions that span multiple lines in JavaScript and TypeScript.",
+          "default": "nextLine",
```

Comment:
Can you verify these defaults are correct? I think some are wrong like this one: https://github.com/dprint/dprint-plugin-typescript/blob/936cf945ad9f0e69bb31cffe1c85b1cf991fe56e/src/configuration/builder.rs#L54

This is actually the default we should have selected, but we did sameLine.

Reply:
Oh, I was using dprint's default. Thanks!

Updated in https://github.com/denoland/deno/pull/28946/commits/ba93914f3d9eef79476726618b55c45150f53c4c